config/samples/default.yaml

```diff
@@ -8,4 +8,24 @@ metadata:
   name: default
   namespace: kyma-system
 spec:
-  # TODO(user): Add fields here
+  labels: {}
+  annotations: {}
+  companionConfig:
+    namespace: ai-core
+    configMapNames:
+      - ai-backend-config
+    secretNames:
+      - ai-core
+      - ai-ghcr-login-secret
+    replicas: 2
```

Comment:
should we allow a min and max of replicas?
```
replicas:
   min: 2
   max: 2
```

Reply:
great idea, let's have it with min and max so that we can use auto scaling.

Reply:
Great idea.